apps/web/src/pages/ApplicantDashboardPage/components/TalentManagementTaskCard.tsx

```diff
@@ -78,6 +83,16 @@ const TalentManagementTaskCard = ({
     },
   ];
 
+  const sortedNominations = unpackMaybes(
+    talentManagementTaskCardFragment.talentNominationsAsSubmitter,
+  )
+    .sort((a, b) => {
+      const aUpdated = a?.updatedAt ? new Date(a.updatedAt) : MAX_DATE;
+      const bUpdated = b?.updatedAt ? new Date(b.updatedAt) : MAX_DATE;
+      return aUpdated.getTime() - bUpdated.getTime();
+    })
+    .sort((a, b) => (a?.submittedAt ? 1 : 0) - (b?.submittedAt ? 1 : 0));
+
```

Comment:
The issue says sort by updated_at, but I'm not so sure about it. Wouldn't closing or submitted date be the more expected sort?

Like, in your example I feel the closer deadline, the one highlighted with warning text should be pushed to the top

![image](https://github.com/user-attachments/assets/65217ebb-0b1b-4554-a60d-be6148682c5c)

Reply:
Yeah, I tihnk that makes a lot of sense. I'll bring it up in stand up just to be sure the whole team feels the same (I'm sure they will :sweat_smile: )

Reply:
Decision in standup: suggestion seems fine though not too important right now.  OK to change or leave.

Reply:
Added in [add closing date to sorting](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13290/commits/6d901ab65c5f2758b1d5d5e46415a595f5c5fc53)